sources/settings-extensions/ntp/Cargo.toml

```diff
@@ -0,0 +1,19 @@
+[package]
+name = "settings-extension-ntp"
+version = "0.1.0"
+authors = ["Sam Berning <bernings@amazon.com>"]
+license = "Apache-2.0 OR MIT"
+edition = "2021"
+publish = false
+
+[dependencies]
+modeled-types = { path = "../../models/modeled-types", version = "0.1" }
```

Comment:
This appears to be unused, and can be removed.

I tend to like [`cargo-machete`](https://github.com/bnjbvr/cargo-machete) for finding cases like these, though it is imprecise.

Reply:
+1 for `cargo machete`, though I've noticed it doesn't recognize relative imports like these. So just a warning.

Reply:
I do use this one for the `Url` type in [lib.rs](https://github.com/bottlerocket-os/bottlerocket/pull/3595/files#diff-34509cee9ac5e2fdc3b51e71de432bbf936201fc7e22a3281f38ef87306f7d2cR5)

Reply:
Whoops! Sorry, I missed that.